docs/website/blog/2022-09-01-odo-v3-rc1.md

```diff
@@ -0,0 +1,111 @@
+tags: ["release"]
+slug: odo-v3-rc1-release
+---
+
+`3.0.0-rc1` of odo has been released!
+
+<!--truncate-->
+
+To install `odo`, follow our installation guide at [odo.dev](../docs/overview/installation)
+
+## Note about Release Candidate
+
+With rc1, we are feature complete with the following exceptions:
+- odo list services ([\#5996](https://github.com/redhat-developer/odo/pull/5996))
+- ability to turn off auto restart and trigger sync manually in odo dev ([\#5634](https://github.com/redhat-developer/odo/pull/5634))
+
+Our GA release will include:
+- Onboarding updates, including updates to the Quickstart guide
+- Video tutorial
+
+## Notable Changes
+
+Check this Playlist for an overview of the most notable changes in this release:
+https://www.youtube.com/watch?v=OM8WjhGhdmw&list=PLGMB2PY4SNOpvdCgJjgp-91Z3wh9eyatV
+
+### Features
+
+#### odo dev and odo deploy now always work on current Kubernetes namespace ([#6025](https://github.com/redhat-developer/odo/pull/6025))
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/OM8WjhGhdmw" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+`odo dev` and `odo deploy` are now always working on the current Kubernetes namespace, and are no longer saving the namespace on which they are working in a local file.
+
+#### odo list: show the component manager version ([#6028](https://github.com/redhat-developer/odo/pull/6028))
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/vJsCink8M-w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+`odo list` now outputs the version of the manager used to deploy a component, as described in the `app.kubernetes.io/managed-by-version` label of the component's deployment.
+
+#### odo list displaying components and bindings ([#6043](https://github.com/redhat-developer/odo/pull/6043))
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/aWxb_GXLW_w" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+`odo list` combines the output of `odo list component` and `odo list binding`, for human-readable and JSON output.
+
+#### odo no longer requires the optional 'metadata.name' field in Devfile ([#6015](https://github.com/redhat-developer/odo/pull/6015))
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/20QOOAwil8M" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+If the `metadata.name` field is present in the Devfile, it is used as name of the deployed component. If not present, the directory name is used instead.
```

Comment:
This needs to be clarified, I think. If `metadata.name` is not present, it uses Alizer to detect name. If not possible, the directory name is used. So maybe we could write it like so:

```suggestion
If the `metadata.name` field is present in the Devfile, it is used as name of the deployed component. If not present, odo tries to detect the name automatically based on the project configuration. If this is not possible, the directory name is used instead.
```